src/main/java/be/syntra/devshop/DevshopBack/entities/Product.java

```diff
@@ -0,0 +1,23 @@
+package be.syntra.devshop.DevshopBack.entities;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+@Table
+@Getter
+@Setter
+@Builder
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "name")
+    private final String name;
+    @Column(name = "price")
+    private final BigDecimal price;
```

Comment:
@Digits(fraction = 2) gaat ervoor zorgen dat onze prijs altijd 2 cijfers na de komma heeft. 
@PositiveOrZero gaat ervoor zorgen dat onze prijs nooit minder is dan 0.